### ISSUE TITLE
HelperText doesn't render correctly when conditionally added

### ISSUE THREAD

**Issue by tuckergordon** [hperrin/svelte-material-ui#627]
**Describe the bug**
Textfield `HelperText` does not render in the correct location if placed inside an `{#if}`. Noticed this with `CharacterCounter` but I'm guessing a similar bug is causing #392 and possibly #245 (though that issue is pretty old at this point). 

I have a hunch it could be related to #384 as well, as those are both situations where an element is conditionally rendered and some of the styles are not loaded correctly.

**To Reproduce**

Add `HelperText` to a `Textfield` inside a conditional, e.g.
```svelte
export let charLimit = 18;

<Textfield label="Write a message..." input$maxLength={charLimit}>
  {#if charLimit}
    <CharacterCounter slot="helper">0 / {charLimit}</CharacterCounter>
  {/if}
</Textfield>
```

**Expected behavior**
Character count displays below textfield

**Screenshots**
<img width="414" alt="image" src="https://github.com/hperrin/svelte-material-ui/assets/7843139/2cd6883c-f76d-467f-9d7f-1416a853c3e0">

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser: Chrome
 - Version `^7.0.0-beta.14`

**Additional context**
A workaround for my use case was to use a `svelte:fragment` so that the slot is always created, and the conditionally add the CharacterCounter:

```svelte
<Textfield label="Write a message..." input$maxLength={charLimit}>
  <svelte:fragment slot="helper">
    {#if charLimit}
      <CharacterCounter>0 / {charLimit}</CharacterCounter>
    {/if}
  </svelte:fragment>
</Textfield>
```

**Comment by hperrin** [comment]
Yeah, Svelte doesn't support conditional slots. The new version supports conditional snippets though.
